apps/projects/app/components/Shared/FilterBar/FilterBar.js

```diff
@@ -25,10 +27,13 @@ const StyledFilterBar = styled.div`
   }
   > :nth-last-child(2) {
     flex: 1 1 auto;
+    z-index: 3;
```

Comment:
Not sure if it's this line but something is pushing the filter bar above the action dropdown in the Issues tab.

Reply:
The FilterBar needs some z-index to work, I updated the dropdown button z-index to fix the issue at 2043d5d